### DISCUSSION TITLE
Exiting with status 11

### DISCUSSION THREAD

**Discussion by obscure1980** [winapps-org/winapps#346] [Support]
Hello Guys

i cant get it to work
my windows container is running but i get always:

Installing WinApps.
Checking for existing conflicting WinApps installations... Done!
Attempting to load WinApps configuration file... Done!
Checking whether dependencies are installed... Done!
Checking container status... Failed!
ERROR: CONTAINER NOT RUNNING.
Windows is not running.

Please ensure Windows is powered on:
docker compose --file ~/.config/winapps/compose.yaml start

if i check wit docker ps everything is fine the container works and i can connect with rdp.
What can i do?

**Comment by SevilleCode** [comment]
I am having the same issue using Docker on Ubuntu 24.04.2. I can visit the IP address and see Windows just fine. The FreeRDP test session connects successfully. So I am really not sure why WinApps is saying that Windows isn't running.

**Comment by BenGrenoble** [comment]
Same issue here I even can launch my apps using : 
xfreerdp /u:user /p:password /cert-ignore /app:'C:\Program Files\Microsoft Office\root\Office16\POWERPNT.EXE' /v:ip

But winapps keeps saying that the container is not running
